simulation/replication/replication_simulation_test.go

```diff
@@ -271,7 +271,7 @@ func changeActiveClusters(
 		updateDomainRequest.ActiveClusterName = &toCluster
 	}
 
-	// TODO(active-active): Remove this once we have completely migrated to AttributeScopes
+	// Migrated from ActiveClustersByRegion to AttributeScopes
 	if op.NewActiveClustersByRegion != nil {
```

Comment:
We can remove this block, because we've set cluster attributes in the test input.

Reply:
if it's ok, let me follow up a PR to remove with this, because I think it'll balloon the PR size beyond the feasible a bit.